lib/datadog/tracing/span_operation.rb

```diff
@@ -352,17 +352,23 @@ class Events
         attr_reader \
           :after_finish,
           :after_stop,
-          :before_start,
-          :on_error
+          :before_start
 
         def initialize(on_error: nil)
```

Comment:
Wait, is there a bug in handling this? This `on_error` never gets used. Was the intent to override the default on error somewhere?

Reply:
`on_error` was used before to expose `@on_error = OnError.new`, created in the constructor.

Now it's defined by the method `def on_error`.